example/storybook/src/components/DataDisplay/Badge/index.stories.mdx

```diff
@@ -421,11 +421,11 @@ We provide in-depth information on how to customize and extend the component's f
 
 ### Customizing the Badge
 
-For customizing the Badge, you have to map styled view to Badge You can refer [style.gluestack.io](https://style.gluestack.io/) for more information on how to use styled components.
+For customizing the Badge, you have to map styled view to Badge You can refer [gluestack.io/style](/style) for more information on how to use styled components.
 
 #### Usage
 
-Default styling of the component can be found in the `components/badge` file. For reference, you can view the [source code](https://github.com/gluestack/gluestack-ui/tree/main/example/storybook/src/ui-components/Badge) of the styled `Badge` component.
+Default styling of the component can be found in the `components/badge` file. For reference, you can view the [source code](https://github.com/gluestack/gluestack-ui/blob/main/packages/themed/src/components/Badge) of the styled `Badge` component.
```

Comment:
https://github.com/gluestack/gluestack-ui/tree/main/packages/themed/src/components/Badge

correct the link